server/src/main/java/org/opensearch/index/query/DisMaxQueryBuilder.java

```diff
@@ -251,8 +251,10 @@ protected void extractInnerHitBuilders(Map<String, InnerHitContextBuilder> inner
     @Override
     public void visit(QueryBuilderVisitor visitor) {
         visitor.accept(this);
+
+        QueryBuilderVisitor qbv = QueryBuilderVisitor.NO_OP_VISITOR;
```

Comment:
Oops -- I'm assuming this line snuck in accidentally.

Reply:
Yeah, I will raise the fix with text cases